src/components/Icon/LikeIcon.tsx

```diff
@@ -0,0 +1,18 @@
+export const LikeIcon = () => (
```

Comment:
Please help to create storybook for all icon

Reply:
Done